src/store/DialogStore.js

```diff
@@ -10,16 +10,18 @@ export default class DialogStore {
 
     @observable open = false;
 
-    @action.bound setOpen(val) {
+    @action.bound setOpen(val , closeActiveDialog = true) {
```

Comment:
Is `closeActiveDialog` just for that overwrite prompt? Doesn't look very appropriate. That prompt should be a dialog by itself.

Reply:
when you open a dilaog it closes the dialog that is alrady opened, i want to open a dialog iniside another, so i tell it not to close the old one(the parent dialog here) , so this propery is basicly tell to not close the activedilaog.

Reply:
aite. Fair point. I'm not sure if this meant we need to keep track of all the active dialogs if we have this setting... Just for one feature it's tedious though.

Reply:
no for the dialog you dont want want  keep track off. like confrim dialogs, in this case if the parrent closes the child that is the confrim dialog will be closed ass well. no need to track it.